pkgs/development/tools/haskell/dconf2nix/dconf2nix.nix

```diff
@@ -1,13 +1,13 @@
-{ mkDerivation, base, containers, fetchgit, hedgehog
-, optparse-applicative, parsec, lib, template-haskell, text
+{ mkDerivation, base, containers, fetchgit, hedgehog, lib
+, optparse-applicative, parsec, template-haskell, text
 }:
 mkDerivation {
   pname = "dconf2nix";
-  version = "0.0.7";
+  version = "0.0.8";
   src = fetchgit {
     url = "https://github.com/gvolpe/dconf2nix.git";
```

Comment:
Please use fetchFromGitHub.

Reply:
This is automatically generated running the [update.sh](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/haskell/dconf2nix/update.sh) script.

Reply:
That script can entirely be replaced with nix-update.

Reply:
Probably, but this PR only intends to update the version, not refactoring everything. That should be done in a different PR IMO.

Reply:
Everything is a bit of a exaggeration. I only asked to use a fitting fetcher. Also fetching commits when there is a tag available is a really bad practice. GitHub mixes commits of all forks and updating with tags through nix-update is just much easier and the current standard how to do such things.

Reply:
Dude. You're asking me to fix `cabal2nix`, which has nothing to do with this PR. The `update.sh` was originally adapted from https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/purescript/spago/update.sh 

If it's so much easier you can surely put up a PR to "do it properly" later on. Now can you please stop the BS and unblock this PR that only updates the version? Thanks.

Reply:
@SuperSandro2000: leaving comments on code that is not touched by the user is counter-productive. As a submitter, I should be allowed to submit a simple version bump and not have my PR blocked by unrelated issues.

For styling issues, it's best to write tools that can automate these fixes.

Reply:
That code is touched by this PR and this is not a styling issue but a workflow issue which causes updates to be unnecessarily hard and none automated. Also fetching commits makes reviews harder because someone needs to check if that commit actually belongs to the repo. When fetching tags this is not necessary.

Reply:
There is a time and place to do things.

If the PR is just a simple version bump, what the contributor cares about is getting the newer version in. It's demotivating when code reviews expand the scope of a PR and delays the merge.

A good metric is to ask yourself if the submitted code is better than the previous code.

You're also right, it would be best if `fetchFromGitHub` and the git tag were being used. For that, the best thing would be to talk to the Haskell team to see if they can add a proper heuristic to cabal2nix.